pkg/utils/k8sutils.go

```diff
@@ -166,3 +166,7 @@ func mutate(f controllerutil.MutateFn, key client.ObjectKey, obj client.Object)
 	}
 	return nil
 }
+
+func GetClusterResourceQuotaName(name string) string {
+	return fmt.Sprintf("storage-client-%s-resourceqouta", GetMD5Hash(name))
```

Comment:
typo in `qouta` its `quota` 

btw why did we need resoucequota in the name as its already ClusterResourceQuota.

Reply:
I cannot change the name of the resource right now because it has inherited from version 4.17. If I change it, I will need to conduct tests for the upgrade and some other scenarios. I will create a new task for this and work on it. Thank you for eyeing on that.